server/README.md

```diff
@@ -27,6 +27,9 @@ The FASTEN-server is a component necessary for running [FASTEN-specific plugins]
     - `-pl` `--plugin_list` List of plugins to run. Can be used multiple times.
     - `-po` `--plugin_output` Path to directory where plugin output messages will be stored
     - `-pol` `--plugin_output_link` HTTP link to the root directory where output messages will be stored
+- `cg` `--consumer_group` Name of the consumer group. Defaults to (canonical) name of the plugin.
+- `ot` `--output_topic` Name of the output topic. Defaults to (simple) name of the plugin.
```

Comment:
The arg. `-kt` already accepts the output topic for plug-ins. So what's the purpose of `ot`?

Reply:
This change was also already implemented in [this PR](https://github.com/fasten-project/fasten/pull/183). `-kt` is not sufficient, because it only sets the consume topic for a plugin (see [here](https://github.com/fasten-project/fasten/blob/a3dc8f3fd29d493df0782bfa132971a1d23d71db/server/src/main/java/eu/fasten/server/FastenServer.java#L235)). Not the output/produce topic. The output topic was always defaulting to  `k.getClass().getSimpleName()`.

Reply:
I see! Perhaps, we need to pass a list like `-kt` for `-ot`? Now, you can only set one output topic assuming that you want to run only one plug-in, which is the case for current deployment.

Reply:
Yes, maybe.. however, what is the chance that FastenServer will ever run more than 1 plugin? I'm not sure if we still want that. I do agree, that we might need to pick one standard and use that for all arguments (either accept for multiple plugins or just for one). I think we should do that in a different PR though